container-search/src/main/java/com/yahoo/search/handler/HttpSearchResponse.java

```diff
@@ -69,14 +69,14 @@ private static void populateHeaders(HeaderFields outputHeaders,
         }
     }
 
-    public ListenableFuture<Boolean> waitableRender(OutputStream stream) throws IOException {
```

Comment:
I don't think we can remove this throw because it breaks the API

Reply:
Hmm, It did not fail the api checks......

Reply:
https://docs.oracle.com/javase/specs/jls/se7/html/jls-13.html#jls-13.4.21
13.4.21. Method and Constructor Throws
Changes to the throws clause of methods or constructors do not break compatibility with pre-existing binaries; these clauses are checked only at compile time.